### PR TITLE
FI-3100: Update (g)(10) Certfication Test Kit Terminology Package to 2024 Version 

### PR DESCRIPTION
**Pull request by emichaud998**
This PR updates the related terminology script code to use the new 2024 UMLS terminology package version. Running the script now will pull down the 2024 UMLS package, and update the (g)(10) Test Kit to use the updated FHIR terminology. This also updates the `lib/inferno/terminology/expected_manifest.yml` file with the new number of values in the updated value sets so that the `bundle exec rake terminology:check_built_terminology` runs without failure. 

Also created instructions for this process and added it as an attachment to the ticket for this task.

### CODE REVIEW

File: lib/inferno/terminology/fhir_package_manager.rb [hunk 1]
```diff
@@ -44,10 +45,14 @@ class << self
         #
         # @param [String] package The FHIR Package
         def get_package(package, destination, desired_types = [])
-          package_url = package
-            .split('#')
-            .prepend(REGISTRY_SERVER_URL)
-            .join('/')
+          package_url = if package.end_with?('#7.0.0')
```
Comment (Jammjammjamm) [inline lib/inferno/terminology/fhir_package_manager.rb:48]: This should be a strict check for the US Core 7 package.

File: lib/inferno/terminology/tasks/download_fhir_terminology.rb [hunk 2]
```diff
@@ -19,8 +19,9 @@ def download_us_core
           FHIRPackageManager.get_package('hl7.fhir.us.core#4.0.0', PACKAGE_DIR, ['ValueSet', 'CodeSystem'])
           FHIRPackageManager.get_package('hl7.fhir.us.core#5.0.1', PACKAGE_DIR, ['ValueSet', 'CodeSystem'])
           FHIRPackageManager.get_package('hl7.fhir.us.core#6.1.0', PACKAGE_DIR, ['ValueSet', 'CodeSystem'])
+          FHIRPackageManager.get_package('hl7.fhir.us.core#7.0.0', PACKAGE_DIR, ['ValueSet', 'CodeSystem'])
           FHIRPackageManager.get_package(
-            'us.nlm.vsac#0.11.0', File.join(PACKAGE_DIR, 'vsac'), ['ValueSet', 'CodeSystem']
+            'us.nlm.vsac#0.17.0', File.join(PACKAGE_DIR, 'vsac'), ['ValueSet', 'CodeSystem']
```
Comment (Jammjammjamm) [inline lib/inferno/terminology/tasks/download_fhir_terminology.rb:24]: This is not the version of the VSAC package referenced in US Core 7.

Comment (Jammjammjamm) [reply lib/inferno/terminology/tasks/download_fhir_terminology.rb]: You'll also have to regenerate everything from scratch.

Comment (Jammjammjamm) [reply lib/inferno/terminology/tasks/download_fhir_terminology.rb]: Run the g10 tests and verify that things look normal compared with what's on prod.

Comment (emichaud998) [reply lib/inferno/terminology/tasks/download_fhir_terminology.rb]: Re-ran with the vsac package update and everything looks normal and works as expected

**Comment by Jammjammjamm** [comment]
Now you have to rerun the terminology generation.

**Comment by emichaud998** [comment]
> Now you have to rerun the terminology generation.

Running now, will let you know if there are any issues

**Comment by emichaud998** [comment]
Ran the terminology generation again and everything looks good and works as expected

**Review by Jammjammjamm — APPROVED** [review]

**Pull request merged**